docs/source/install.rst

```diff
@@ -120,55 +101,39 @@ an SSL error unless you disable the SSL verification:
         binstar config --set ssl_verify False
 
 
-4. Once conda and mamba are properly working, you can install the **(a) Latest Stable Release** or
+4. Once conda are properly working, you can install the **(a) Latest Stable Release** or
 create a **(b) Development Environment**.
 
 .. _install_latest:
 
 (a) Latest Stable Release
 -------------------------
 
-1. Follow :ref:`"Others/Local" <conda_environment_others>` section for installing Conda.
-
-2. Get the yml file to create an environment.
-
-    ::
-
-        wget https://raw.githubusercontent.com/E3SM-Project/e3sm_diags/main/conda-env/prod.yml
-
-
-3. Change ``prefix`` in that file to be your conda prefix. Typically, this will be ``~/miniconda3/envs/e3sm_diags_env``.
+1. Follow :ref:`"Others/Local" <conda_environment_others>` section for installing conda.
 
-4. Remove any cached conda packages. This will ensure that you always get the latest packages
+Create a new conda environment with ``e3sm_diags`` installed and activate it:
 
     ::
 
-        mamba clean --all
-
-5. Use conda to create a new environment with E3SM Diags (``e3sm_diags``) included.
-These steps should not be necessary if you installed Mambaforge as suggested
-above but may be needed if you have previously installed Miniconda3 instead: ::
-
-   conda install -y -n base mamba
-   conda config --add channels conda-forge
-   conda config --set channel_priority strict
-
-Create a new conda environment with ``e3sm_diags`` installed and activate it: ::
-
-        mamba env create -f conda-env/prod.yml # Tip: Add the flag ``-n <name_of_env>`` to customize the name of the environment  
+        # Tip: Add the flag ``-n <name_of_env>`` to customize the name of the environment
+        conda create -n e3sm_diags_env e3sm_diags
         conda activate e3sm_diags_env
 
 .. _dev-env:
```

Comment:
Replace the "a) Latest Stable Release" section with just creating a fresh conda env with latest stable `e3sm_diags`.